great-idea-website/css/index.css

```diff
@@ -63,4 +63,72 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+/*ENTIRE PAGE*/
+.container {
+    margin:auto;
+	width: 900px;
+	text-align: left;
+	}
+
+/*NAV @ TOP OF PAGE*/
+header nav {
+	display: inline-block;
+	margin: 50px 0 0 0;
+}
+/*GREAT IDEA LIGHTBULB*/
+header .logo{
+	margin: -25px 140px 2px 637px;
+
+}
+header nav a {
+		  /*TB LR*/
+	margin: 0 20px; 
+	color: gray;
+	text-decoration: none;
+}
+/*INOVATION ON DEMAND TITLE*/
+.innovte{
+	margin-top: 100px;
+    margin-left: 100px;
+    font-family: 'Bangers', cursive;
+	letter-spacing: 1px;
+	font-size: 100px;
```

Comment:
font size might be a bit too big which is partially why your IOD is hiding behind your logo.